### PR TITLE
chore: improve logs

### PR DESCRIPTION
**Pull request by Udit-takkar**
- Improve some logs and replaced stringify with safeStringify

https://github.com/calcom/cal.com/pull/12460#discussion_r1400414241

### CODE REVIEW

**Review by PeerRich — DISMISSED** [review]
good refactor

File: packages/app-store/googlecalendar/lib/CalendarService.ts [hunk 1]
```diff
@@ -106,6 +107,7 @@ export default class GoogleCalendarService implements Calendar {
         });
         myGoogleAuth.setCredentials(googleCredentials);
       } catch (err) {
+        this.log.error("Error Refreshing Google Token", safeStringify(err));
```
Comment (Udit-takkar) [inline packages/app-store/googlecalendar/lib/CalendarService.ts:110]: Added this log to check why we are unable to refresh token for some users.

**Review by keithwillcode — APPROVED** [review]

**Review by keithwillcode — DISMISSED** [review]
Lot's of tests are failing, @Udit-takkar

**Comment by Udit-takkar** [comment]
> Lot's of tests are failing, @Udit-takkar

Getting this Error: Docker pull failed with exit code 1

File: packages/app-store/googlecalendar/lib/CalendarService.ts [hunk 2]
```diff
@@ -251,7 +253,10 @@ export default class GoogleCalendarService implements Calendar {
         iCalUID: event.data.iCalUID,
       };
     } catch (error) {
-      console.error("There was an error contacting google calendar service: ", error);
+      console.error(
```
Comment (sean-brydon) [inline packages/app-store/googlecalendar/lib/CalendarService.ts:256]:
```suggestion
      this.log.error(
```

**Review by sean-brydon — COMMENTED** [review]
LGTM - will be superhelpful debugging 

lets just get this one nit changed then i we can merge

**Review by sean-brydon — APPROVED** [review]
LGTM

**Review by ThyMinimalDev — APPROVED** [review]
Looking good !

**Pull request merged**